tuning/config/configs.py

```diff
@@ -122,6 +126,20 @@ class DataArguments:
             Passed in conjunction with response_template"
         },
     )
+    text_field_name: str = field(
+        default=None,
+        metadata={
+            "help": "Required for running with vision models. \
+            The column name of the text data in the multi-modal dataset."
+        },
+    )
+    image_field_name: str = field(
```

Comment:
Will comment on these fields in the data collator section -- questions around how to load a multimodal dataset that contains images and text and how to preprocess the text portions